types/codemirror/index.d.ts

```diff
@@ -916,7 +916,7 @@ declare namespace CodeMirror {
 
         /** Gets the mode object for the editor. Note that this is distinct from getOption("mode"), which gives you the mode specification,
         rather than the resolved, instantiated mode object. */
-        getMode(): any;
+        getMode(): Mode<any>;
```

Comment:
This change will conflict with #52312 which overhauls the whole codemirror types (but actually documents that as `Mode<unknown>` as that's more accurate than `Mode<any>`)

Reply:
Good point about `unknown` being a better fit here than `any`. I mostly made this change so that I could have a "test", though I don't consider that one to be too valuable (since there's no fully-featured mode available as far as I can tell, so the amount of testing I can do is limited...). How about instead I drop the useless test and the getMode() change, and submit just the indent() change?

Reply:
yeah, I think that's probably the best way to go to avoid conflicts.

Reply:
Done (amended the original commit).